tests/integration_tests/test_hyperopt_ray_horovod.py

```diff
@@ -1,306 +1,306 @@
-# Copyright (c) 2019 Uber Technologies, Inc.
```

Comment:
I can't approve merging this file commented out without explanation or documentation.  What would we need to do to un-comment this?  Is it fixed in another PR?

Reply:
I'll be uncommenting this out once I have a fix in place for this file. This was just commented out to make sure the tests pass.